docs/src/pages/demos/progress/LinearBuffer.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import LinearProgress from '@material-ui/core/LinearProgress';
+
+const useStyles = makeStyles({
+  root: {
+    flexGrow: 1,
+  },
+});
+
+function LinearBuffer() {
+  const classes = useStyles();
+  const [completed, setCompleted] = React.useState(0);
+  const [buffer, setBuffer] = React.useState(10);
+
+  const progress = React.useRef<() => void>(() => undefined);
```

Comment:
Does
```suggestion
  const progress = React.useRef(() => {});
```

work as well?

From my experience people prefer less type syntax. While I don't always agree with that I see no harm here. Type of the ref is apparent from the initial value.

Reply:
@eps1lon that does work! I committed your suggestion!